disnake/components.py

```diff
@@ -279,6 +276,7 @@ def __init__(self, data: SelectMenuPayload):
         self.options: List[SelectOption] = [
             SelectOption.from_dict(option) for option in data.get("options", [])
         ]
+        self.values: List[str] = data.get("values", [])
```

Comment:
I believe this can be removed now that #583 is merged

Reply:
https://github.com/DisnakeDev/disnake/pull/558/commits/31d2e119bdf7c5597c4d3af2f41dc0e3d4f6eee3